checker/src/main/java/org/checkerframework/checker/formatter/FormatterVisitor.java

```diff
@@ -88,9 +89,14 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
                                     default:
                                         if (!fc.isValidParameter(formatCat, paramType)) {
                                             // II.3
+                                            ExecutableElement method =
+                                                    TreeUtils.elementFromUse(node);
+                                            Name methodName = method.getSimpleName();
                                             tu.failure(
                                                     param,
                                                     "argument.type.incompatible",
+                                                    "", // parameter name is not useful
```

Comment:
I don't see a simple way around this, but this will result in a message like

```
incompatible argument for parameter  of foo
```

with two spaces between `parameter` and `of`. (Interestingly, the markup doesn't show two spaces with inline code...)

Reply:
You are right.  I didn't see a good way around it either.  It would be possible to fix by passing a single extra argument with text "parameter arg of foo" or "parameter of foo", but to me that looked uglier (in the source code) than leaving an extra space (in the output).  Let's fix it if a user complains.